dfsclient/src/test/java/org/hpccsystems/dfs/client/DFSReadTest.java

```diff
@@ -57,11 +47,12 @@ public void setup()
             espConn.setUserName("");
             espConn.setPassword("");
 
-            this.file = new HPCCFile(testIntegerKVDataset, espConn);
-            this.file.setProjectList("key");
+            this.file = new HPCCFile(testIntegerKVDataset, espConn)
+                    .setProjectList("key")
+                    .setFilter(testDatasetFilterExpr);
 
-            this.file2 = new HPCCFile(testAllTypesDataset, espConn);
-            this.file2.setProjectList("str,qstr,uni10,bool,childdataset.childindex,childdataset.childstr");
+            this.file2 = new HPCCFile(testAllTypesDataset, espConn)
+                    .setProjectList("str,qstr,uni10,bool,childdataset.childindex,childdataset.childstr");
```

Comment:
is this the style we're going with for method chaining when it would go past 80 characters in length?  Is this consistent elsewhere in the project?

Reply:
The format template wraps at 150 or so chars, I chose to wrap earlier to emphasize what this test is exercising.

Reply:
Gotcha.  Looks good then.